src/main/java/echoserver/CommandLinePortValidator.java

```diff
@@ -0,0 +1,23 @@
+package echoserver;
+
+import java.util.regex.Pattern;
+
+public class CommandLinePortValidator {
+    public CommandLinePortValidator() {
+    }
+
+    public Integer parsePort(String[] command) {
+        if (validate(command)) {
+            return Integer.parseInt(command[1]);
+        }
+
+        return 5000;
```

Comment:
I wonder if the default (5000) should be a constructor argument.

Reply:
It looks like this one may have been overlooked. Just wanted to bring it to your attention.